pycbc/events/stat.py

```diff
@@ -198,52 +210,82 @@ def coinc(self, s0, s1, slide, step): # pylint:disable=unused-argument
         cstat: numpy.ndarray
             Array of coincident ranking statistic values
         """
-        cstat = (s0**2. + s1**2.) ** 0.5
-        cstat[s0==-1] = 0
-        cstat[s1==-1] = 0
+        cstat = (s0 ** 2. + s1 ** 2.) ** 0.5
+        cstat[s0 == -1] = 0
+        cstat[s1 == -1] = 0
         return cstat
 
 
 class PhaseTDStatistic(NewSNRStatistic):
-
     """Statistic that re-weights combined newsnr using coinc parameters.
 
     The weighting is based on the PDF of time delays, phase differences and
     amplitude ratios between triggers in different ifos.
     """
-    def __init__(self, files):
-        NewSNRStatistic.__init__(self, files)
-        self.hist = self.files['phasetd_newsnr']['map'][:]
-
-        # Normalize so that peak has no effect on newsnr
-        self.hist = self.hist / float(self.hist.max())
-        self.hist = numpy.log(self.hist)
 
-        # Bin boundaries are stored in the hdf file
-        self.tbins = self.files['phasetd_newsnr']['tbins'][:]
-        self.pbins = self.files['phasetd_newsnr']['pbins'][:]
-        self.sbins = self.files['phasetd_newsnr']['sbins'][:]
-        self.rbins = self.files['phasetd_newsnr']['rbins'][:]
+    def __init__(self, files, ifos):
```

Comment:
ifos should still be optional here to not break the two-ifo workflow and be consistent with the Stat class definition.

Reply:
ok .. yes, this makes sense, I did test the 2-ifo case but probably some 'semi-random' combination of defaults saved it from erroring